src/screens/Home/TransferScreen.js

```diff
@@ -151,10 +153,13 @@ class TransferScreen extends React.Component {
 
 const mapStateToProps = ({ payutc }) => {
 	const suggestions = payutc.getUserAutoComplete();
+	const history = payutc.getHistory();
 
 	return {
 		suggestions: suggestions.getData([]),
 		suggestionsFetching: suggestions.isFetching(),
+		history: history.getData({ historique: [] }).historique,
```

Comment:
Here, you define that your `history` key has for value `historique`. The API send always an array. While it is not fetched, `getData` will return the default value which is `{ historique: [] }`. So, in all cases, `historique` is an array with a length >= 0